extensions/wikia/Search/WikiaSearchController.class.php

```diff
@@ -881,7 +881,8 @@ protected function templateExists( $name ) {
 	 * @see SearchControllerTest::testIsCorporateWiki
 	 */
 	protected function isCorporateWiki() {
-		return WikiaPageType::isCorporatePage();
+		// TODO: refactor to avoid using wgEnableWikiaHomePageExt
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this "TODO" comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1135)